app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
```

Comment:
Based on the wireframes, it doesn't actually seem like we need this action, so I would remove it and the associated view.